packages/jest-resolve/src/ModuleNotFoundError.ts

```diff
@@ -31,11 +31,11 @@ export default class ModuleNotFoundError extends Error {
 
     let message = this._originalMessage;
 
-    if (this?.requireStack?.length && this!.requireStack!.length > 1) {
+    if (this.requireStack?.length && this.requireStack.length > 1) {
```

Comment:
👍

Reply:
```suggestion
    if (this.requireStack?.length > 1) {
```

?

Reply:
`>` requires both sides to be a number, so you can't do `undefined > 1` in TS - I thought about merging these but decided against it as I think the alternatives don't really add much while not being as clean, i.e `(this.requireStack?.length ?? 0) > 1`